doc/source/whatsnew/v2.0.0.rst

```diff
@@ -291,6 +291,39 @@ The new behavior, as for datetime64, either gives exactly the requested dtype or
    ser.astype("timedelta64[s]")
    ser.astype("timedelta64[D]")
 
+.. _whatsnew_200.api_breaking.zero_len_indexes:
+
+Empty DataFrames/Series will now default to have a ``RangeIndex``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When users don't specify indexes or columns when instantiating :class:`DataFrame` or :class:`Series`
```

Comment:
Saw this in some tests: Based on `users don't specify indexes or columns` does e.g. `Series(index=[])` also now produce a `RangeIndex`?

Reply:
I only changed the case where `index=None`, so `Series(index=[]).index` is unchanged an `Index` with dtype=object. IMO `index=[]`should be inferred to `index=Index([])`...

Reply:
Okay gotcha. Could you then clarify here that `users don't specify indexes or columns` explicitly means when `index/columns=None`